lib/controller/messagescontroller.php

```diff
@@ -153,6 +153,10 @@ public function show($accountId, $folderId, $id) {
 			}, $json['attachments']);
 		}
 
+		// Add accountId, folderId for unified inbox replies
+		$json['accountId'] = $accountId;
```

Comment:
``$accountId`` is still set to ``-1`` for the unified inbox. @DeepDiver1975 how do I get the real accountId here? 

Same goes for ``$folderId``.